packages/terra-application-layout/docs/responsiveDesign.md

```diff
@@ -0,0 +1,32 @@
+# Terra Application Layout
+
+## Responsive Design
+
+The ApplicationLayout has two rendering modes: `standard` and `compact`.
+  - The `standard` rendering occurs at `medium`, `large`, and `huge` breakpoints.
+  - The `compact` rendering occurs at `tiny` and `small` breakpoints.
+
+### Header
+
+The ApplicationLayout header has a number of changes between its `standard` and `compact` rendering modes.
+
+- `nameConfig`
```

Comment:
I feel listing this is as a table might be easier to articulate the prop rendering differences

Reply:
I agree, I'm making more tables and listening to this: https://www.youtube.com/watch?v=adyC404_gEk